server/graphql/location/src/lib.rs

```diff
@@ -59,6 +62,60 @@ impl LocationQueries {
             locations,
         )))
     }
+
+    /// Returns a list of locations which are available to be assigned to assets
+    pub async fn asset_locations(
```

Comment:
```suggestion
    pub async fn unassigned_asset_locations(
```

Not sure what the best wording is but asset_locations sounds like it would return the assets assigned to an asset, not the ones that are not assigned.

Not a big deal but I'd make it more specific...

Reply:
I've created a suggestion branch, where we can just pass a boolean into the Location Filter, instead of this specific query. 
Might be a nice way to go and more flexible for future use cases? https://github.com/msupply-foundation/open-msupply/compare/3462-add-cce-column...3462-suggestion_subquery